h/static/scripts/group-forms/utils/test/annotation-metadata-test.js

```diff
@@ -0,0 +1,101 @@
+import { pageLabel, quote, username } from '../annotation-metadata';
+
+describe('quote', () => {
+  it('returns quote if annotation has a quote', () => {
+    const ann = {
+      target: [
+        {
+          source: 'https://publisher.org/article.pdf',
+          selector: [{ type: 'TextQuoteSelector', exact: 'expected quote' }],
+        },
+      ],
+    };
+    assert.equal(quote(ann), 'expected quote');
+  });
+
+  // FIXME - This currently happens when creating a new Page Note. Annotations
+  // from the API should always have a target.
```

Comment:
In this context we're _always_ working with annotations from the API. In the client we should just fix this so the `target` is always a non-empty array.